tests/functional/TestKubeSmokeTest/testHeketiRpc.sh

```diff
@@ -26,12 +26,35 @@ display_information() {
 	kubectl get nodes
 }
 
+create_fake_application() {
+	pod=$1
+	app=$2
+	kubectl exec $pod -- sh -c "echo '#!/bin/sh' > /bin/${app}" || fail "Unable to create /bin/${app}"
+	kubectl exec $pod -- chmod +x /bin/${app} || fail "Unable to chmod +x /bin/${app}"
+}
+
+create_bash() {
+	pod=$1
+	app="bash"
+	kubectl exec $pod -- sh -c "echo '#!/bin/sh' > /bin/${app}" || fail "Unable to create /bin/${app}"
+	kubectl exec $pod -- sh -c "echo 'sh \$@' >> /bin/${app}" || fail "Unable to add to /bin/${app}"
+	kubectl exec $pod -- chmod +x /bin/${app} || fail "Unable to chmod +x /bin/${app}"
+}
+
+create_fake_vgdisplay() {
+	pod=$1
+	app=vgdisplay
+	kubectl exec $pod -- sh -c "echo '#!/bin/sh' > /bin/${app}" || fail "Unable to create /bin/${app}"
+	kubectl exec $pod -- sh -c "echo 'echo mock:r/w:772:-1:0:2:2:-1:0:1:1:249278464:4096:60859:60859:0:FcehVp-rc3l-xTBH-ZGxK-53G2-GrOr-bDQzQF' >> /bin/${app}" || fail "Unable to add to /bin/${app}"
```

Comment:
Sure.  Here we fake the return of a `vgdisplay` command as needed by https://github.com/heketi/heketi/blob/master/executors/sshexec/device.go#L98 .  For this to succeed, Heketi must find the pod, execute the command inside the container, get the output, and parse it.